Dockerfile

```diff
@@ -4,9 +4,17 @@ FROM google/dart-runtime-base:1.23.0
 # docker image diff small.
 RUN apt-get update && \
     apt-get upgrade -y && \
-    apt-get install -y memcached && \
+    apt-get install -y git memcached unzip && \
     rm -rf /var/lib/apt/lists/*
 
+ENV PATH="/flutter/bin:${PATH}"
+# To ensure actions from this container are not seen as "typical" users.
```

Comment:
Maybe rephrase this a bit:

`Let the pub server know that this is not a "typical" pub client but rather a bot.`